net-im/corebird/corebird-1.5.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+VALA_MIN_API_VERSION=0.28
+
+AUTOTOOLS_AUTORECONF=1
+AUTOTOOLS_IN_SOURCE_BUILD=1
+inherit eutils autotools-utils gnome2 vala
+
+DESCRIPTION="Native GTK+3 Twitter client"
+HOMEPAGE="http://corebird.baedert.org/"
+SRC_URI="https://github.com/baedert/corebird/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug gstreamer spellcheck"
+
+RDEPEND="dev-db/sqlite:3
+	>=dev-libs/glib-2.44:2
+	dev-libs/json-glib
+	gstreamer? ( media-plugins/gst-plugins-meta:1.0[X]
+		media-libs/gst-plugins-base:1.0[X]
+		>=media-libs/gst-plugins-bad-1.6:1.0[X]
+		media-libs/gst-plugins-good:1.0
+		media-plugins/gst-plugins-libav:1.0
+		media-plugins/gst-plugins-soup:1.0
+		media-plugins/gst-plugins-hls:1.0 )
+	spellcheck? ( >=app-text/gspell-1.0 )
+	>=net-libs/libsoup-2.42.3.1
+	>=net-libs/rest-0.7.91:0.7
+	>=x11-libs/gtk+-3.18:3"
+DEPEND="${RDEPEND}
+	$(vala_depend)
+	>=dev-util/intltool-0.40
+	sys-apps/sed
+	virtual/pkgconfig"
+
+src_prepare() {
+	epatch "${FILESDIR}/${PN}-fix-am-prog-valac-0.34.patch"
```

Comment:
Was this upstreamed? Is it needed only for vala < 0.34? If so, it might just make sense to bump the dep and forget about old vala versions. They're on their way to being removed anyway (#4552).